### DISCUSSION TITLE
Π2020158 | Παραδοτέο 5

### DISCUSSION THREAD

**Discussion by mariakourti** [courses-ionio/sw#1579] [Show and tell]
Σε αυτό το παραδοτέο διάλεξα ως κύρια θέματα τα προγράμματα Blender και Unreal Engine 5, αφού με ενδιαφέρει ο τομέας της διασκέδασης και συγκεκριμένα των video games.

[Blender Gallery Link](https://github.com/mariakourti/_gallery/blob/master/blender.md) | [Unreal 5 Gallery Link](https://github.com/mariakourti/_gallery/blob/master/unreal-5.md)

[Blender Image Link](https://github.com/mariakourti/images/blob/master/blender.png) | [Blender Thumb Image Link](https://github.com/mariakourti/images/blob/master/blender-thumb.png)

[Unreal 5 Image Link](https://github.com/mariakourti/images/blob/master/unreal-5.jpg) | [Unreal 5 Thumb Image Link](https://github.com/mariakourti/images/blob/master/unreal-5-thumb.jpg)

[Blender Netlify Link](https://p2020158.netlify.app/gallery/blender) | [Unreal 5 Netlify Link](https://p2020158.netlify.app/gallery/unreal-5)

[Slides Github Link](https://github.com/mariakourti/site/blob/%CE%A02020158/_slides/tools.md) | [Timeline Github Link](https://github.com/mariakourti/site/blob/%CE%A02020158/_timeline/softwaretools.md)

[Slides Netlify Link](https://p2020158.netlify.app/slides/tools) | [Timeline Netlify Link](https://p2020158.netlify.app/timeline/softwaretools)

[Η Αναφορά μου](https://github.com/mariakourti/sw/tree/2020158/projects/2020158)

**Comment by Alessenia** [comment]
Καλησπέρα Μαρία, στο link του Unreal 5 image , έχεις βάλει το link του blender-thumb, αν θέλεις άλλαξε το!

**Reply by mariakourti** [reply]
ευχαριστώω
